### PR TITLE
[SDPA-910] Add padding to Display Heading typography.

### PR DESCRIPTION
**Pull request by alan-cole**
**JIRA issue:** https://digital-engagement.atlassian.net/browse/SDPA-910

### Changed

* Add padding to Display Heading typography.
* Update typography function to optionally include base rules (rules that are only needed once for all breakpoints).

### Screenshots

Screen Shot 2019-08-08 at 2 51 11 pm

### CODE REVIEW

File: packages/components/Atoms/Global/scss/tools/_typography.scss [hunk 1]
```diff
@@ -9,17 +9,32 @@
...
 }
 
-@mixin rpl_display_background($font-size) {
-  background-color: rpl-color('secondary');
-  color: rpl-color('white');
-  display: inline;
+@mixin rpl_display_background($font-size, $include-base: true) {
+  @if $include-base == true {
+    background-color: rpl-color('secondary');
+    color: rpl-color('white');
```
Comment (tim-yao) [inline packages/components/Atoms/Global/scss/tools/_typography.scss]: Let's use this PR to move hardcoded color out of this mixin, it's hard to customised. I will suggest to use sass variables like `$rpl-display-background-color` and `$rpl-display-text-color`.
We should never use `rpl-color` directly, always use variable.

File: packages/components/Atoms/Global/scss/tools/_typography.scss [hunk 2]
```diff
@@ -1,4 +1,4 @@
-@mixin rpl_typography_font($font-size: 's', $line-height: 1.2em, $font-family: 'inherit') {
+@mixin rpl_typography_font($font-size: 's', $line-height: 1.2em, $font-family: 'inherit', $include-base: true) {
```
Comment (tim-yao) [inline packages/components/Atoms/Global/scss/tools/_typography.scss]: Hi @alan-cole Any purpose for `include-base` parameter? Looks like no place is not using it?

**Comment by tim-yao** [comment]
Also, can we add a notes in somewhere docs, stories, for no IE11/Edge support on padding?

File: packages/components/Atoms/Global/scss/tools/_typography.scss (hunk 1 above)
Comment (alan-cole) [reply packages/components/Atoms/Global/scss/tools/_typography.scss]: Will fix.

File: packages/components/Atoms/Global/scss/tools/_typography.scss (hunk 2 above)
Comment (alan-cole) [reply packages/components/Atoms/Global/scss/tools/_typography.scss]: This is for line 63 - 68 of this file:

> Avoid unnecessary duplication; include base styles only on first breakpoint.

When we use the breakpoint loop in rpl_typography_ruleset() for adding styles; only the font-family / line-height / font-size change, but we are also repeating all the other rules as well. This cuts down on some of that unnecessary duplication.

Comment (tim-yao) [reply packages/components/Atoms/Global/scss/tools/_typography.scss]: @alan-cole that makes sense, thanks!

**Pull request merged**